### DISCUSSION TITLE
Want to update contentCompatibility for Questionset

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-Knowlg/Community#175] [Q&A] [answered]
Hi,
I'm trying to fix a bug where an [error happens in the Telemetry while previewing a Question set](https://project-sunbird.atlassian.net/browse/IQ-567).
I figured out that the change needs to happen in a file called ['error.service.ts'](https://github.com/project-sunbird/sunbird-player-sdk/blob/master/projects/sunbird-player-sdk/src/lib/player-utils/service/error/error.service.ts#L7) in the 'sunbird-player-sdk' project.

I want to change the 'playerContentCompatibiltyLevel' to 6 for the Question set. But if I make this change from 5 to 6, I'm worried it might mess things up for other content types. Can you confirm if my concern is valid?

My idea is to create a new variable called 'playerQuestionsetCompatabilityLevel' specifically for checking compatibility with Question sets. That way, it won't affect other content. What do you think?

To make this change, I will be needing a separate branch where I can work on the fix.

cc: 
@pallakartheekreddy @HarishGangula @swayangjit @vinukumar-vs @rajeevsathish 
@sajeshkayyath @PoojaJManjooran @vaibhavbhuva @krgauraw

**Comment by HarishGangula** [comment, marked as answer]
@rajnishdargan 
Instead of adding a new variable, we can override the `playerContentCompatibiltyLevel` while importing,  i mean you can update the module import to take config similar to how the angular core router has `forROOT`, `forChild` and override the default value with whatever is required.

I have created `release-6.0.0` branch, You can work on that branch from your fork.
